src/hammer-vlsi/test.py

```diff
@@ -1051,6 +1051,120 @@ def step5(x: hammer_vlsi.HammerTool) -> bool:
                 else:
                     self.assertFalse(os.path.exists(file))
 
+class HammerSubmitCommandTestContext:
```

Comment:
Comment: this seems to duplicate code with HammerToolHooksTestContext. We don't necessarily need to fix it urgently, although it might be nice to consolidate at some point (large amounts of copy pasted code can get hairy as we all know).

Reply:
Agree

Reply:
Opened #231